R/create_service_use_lookup.R

```diff
@@ -0,0 +1,775 @@
+#' Create the Service Use Cohort lookup file
+#'
+#' @param data A data frame
+#' @param year The financial year
+#' @param write_to_disk Default `TRUE`, will write the lookup to the
+#' Cohorts folder defined by [get_slf_dir]
+#'
+#' @return The Service Use lookup file
+#' @export
+#'
+#' @family Demographic and Service Use Cohort functions
+create_service_use_lookup <- function(data, year, write_to_disk = TRUE) {
```

Comment:
```suggestion
create_service_use_cohort <- function(data, year, write_to_disk = TRUE) {
```
One small name change suggestion as i said in the demographic cohort script - we may get confused with the social care demographic lookup. Suggesting this one for consistency

Reply:
```suggestion
create_service_use_cohorts <- function(data, year, write_to_disk = TRUE) {
```